.circleci/config.yml

```diff
@@ -119,263 +239,70 @@ jobs:
-      - store_artifacts:
-          path: cypress/videos
-      - store_artifacts:
-          path: cypress/screenshots
-
-  detect-secrets:
-    docker:
-      - image: artsy/detect-secrets:ci # pragma: allowlist secret
-    resource_class: small
-    working_directory: /usr/src/app
-    steps:
-      - checkout
-      - run: git ls-files -z | xargs -0 detect-secrets-hook --baseline .secrets.baseline --verbose
-
-  smoke-test-on-live-env:
-    docker:
-      - image: cypress/included:12.17.4
-        environment:
-          TERM: xterm
-    working_directory: ~/app
-    parallelism: 6
-    steps:
-      - checkout
-      - run:
-          name: Install Dependencies
-          command: yarn install
-      - run: |
-          TESTS=$(circleci tests glob "cypress/e2e/**/*" | circleci tests split | paste -sd ',')
-          ./node_modules/.bin/cypress run --config baseUrl=$SMOKE_TEST_TARGET --spec $TESTS
-      - store_artifacts:
-          path: cypress/screenshots
-
-  bundle-stats:
-    docker:
-      - image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/force:$CIRCLE_SHA1-builder
-        aws_auth:
-          aws_access_key_id: $AWS_ACCESS_KEY_ID
-          aws_secret_access_key: $AWS_SECRET_ACCESS_KEY
-    working_directory: /app
-    steps:
-      - run: yarn bundle-report:client --json webpack-stats.json
-      - run: npx relative-ci-agent
-
-not_main_or_staging_or_release: &not_main_or_staging_or_release
-  filters:
-    branches:
-      ignore:
-        - main
-        - staging
-        - release
-
-not_staging_or_release: &not_staging_or_release
-  filters:
-    branches:
-      ignore:
-        - staging
-        - release
-
-only_main: &only_main
-  context:
-    - hokusai
-    - slack-orb
-  filters:
-    branches:
-      only: main
-
-only_release: &only_release
-  context:
-    - hokusai
-    - slack-orb
-  filters:
-    branches:
-      only: release
-
 workflows:
   default:
     jobs:
       - detect-secrets:
           <<: *not_main_or_staging_or_release
 
-      # Main build
-      - artsy-remote-docker/buildkit-build:
-          <<: *not_staging_or_release
-          context: hokusai
-          executor: hokusai/beta
-          name: builder-image-build
-          pre-steps:
-            - run:
-                command: echo 'export BUILD_TARGET="builder";' >> $BASH_ENV
-
-      - artsy-remote-docker/buildkit-push:
-          <<: *not_staging_or_release
-          context: hokusai
-          executor: hokusai/beta
-          name: builder-image-push
-          requires:
-            - builder-image-build
-          pre-steps:
-            - run:
-                command: echo 'export BUILD_TARGET="builder";' >> $BASH_ENV
+      # Danger
+      - node/run:
+          <<: *not_main_or_staging_or_release
+          context:
+            - danger-github-oss
+          name: check-pr
+          pkg-manager: yarn
+          yarn-run: danger:ci
+          version: 18.15.0
```

Comment:
Note `node/run` as an official replacement for `artsy/yarn`, running node scripts, and caching. Whenever we encounter `artsy/yarn` we should remove!